gstd/src/msg/basic.rs

```diff
@@ -474,3 +474,20 @@ pub fn source() -> ActorId {
 pub fn value() -> u128 {
     gcore::msg::value()
 }
+
+pub fn create_program_wgas<T1: AsRef<[u8]>, T2: AsRef<[u8]>>(
+    code_hash: CodeHash,
+    salt: T1,
+    payload: T2,
+    gas_limit: u64,
+    value: u128,
+) -> ActorId {
+    gcore::msg::create_program_wgas(
```

Comment:
considering about where should this method be

is `msg` module is suitable enough for that logic?

Reply:
well, basically, we send `init` message, so seems that should in messages module. What do you think?

Reply:
decided to move to `prog`